src/Metrics.Extensions/src/Public/Microsoft.ApplicationInsights.Metrics/MetricSeriesConfigurationForAccumulator.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+using Microsoft.ApplicationInsights.Metrics.Extensibility;
+using Microsoft.ApplicationInsights.Metrics.Extensions;
+
+namespace Microsoft.ApplicationInsights.Metrics
+{
+    /// <summary>
+    /// 
```

Comment:
I'm missing these description. This code is far from being self-descriptive.

Reply:
We are still in the extensions assembly